### ISSUE TITLE
failed to load sources-jar for jitpack dependency

### ISSUE THREAD

**Issue by emanresusername** [lihaoyi/Ammonite#623]
this might belong with coursier or jitpack, but i'm leaning towards here

with a minimal `predef.sc` 
```scala
interp.repositories() ++= Seq(
    coursier.maven.MavenRepository("https://jitpack.io")
)

val ammoniteGroup = s"ammonite-shell_${scala.util.Properties.versionNumberString}"

Seq(
  "com.lihaoyi"                 % ammoniteGroup         % ammonite.Constants.version,
  "com.github.emanresusername"  % "squants-fx"          % "1e0e64ffbd"
).foreach(interp.load.ivy(_))
@
val shellSession = ammonite.shell.ShellSession()
import shellSession._
import ammonite.ops._
import ammonite.shell._
ammonite.shell.Configure(repl, wd)
```
i'm getting this error
```text
Loading...
Compiling ~/.ammonite/predef.sc
Exception in thread "main" java.lang.RuntimeException: Error during Predef:
        at ammonite.interp.PredefInitialization$.$anonfun$apply$7(PredefInitialization.scala:68)
        at ammonite.interp.PredefInitialization$.$anonfun$apply$7$adapted(PredefInitialization.scala:48)
        at scala.collection.TraversableLike$WithFilter.$anonfun$map$2(TraversableLike.scala:739)
        at scala.collection.immutable.List.foreach(List.scala:389)
        at scala.collection.TraversableLike$WithFilter.map(TraversableLike.scala:738)
        at ammonite.interp.PredefInitialization$.apply(PredefInitialization.scala:48)
        at ammonite.interp.Interpreter.<init>(Interpreter.scala:103)
        at ammonite.repl.Repl.<init>(Repl.scala:44)
        at ammonite.Main.instantiateRepl(Main.scala:86)
        at ammonite.Main.run(Main.scala:134)
        at ammonite.Main$.$anonfun$main$1(Main.scala:282)
        at scala.Option.foreach(Option.scala:257)
        at ammonite.Main$.main(Main.scala:260)
        at ammonite.Main.main(Main.scala)
Caused by: java.lang.Exception: Failed to load dependencies
  not found: https://jitpack.io/com/github/emanresusername/squants-fx/1e0e64ffbd/squants-fx-1e0e64ffbd-sources.jar
        at ammonite.interp.Interpreter$DefaultLoadJar.ivy(Interpreter.scala:547)
        at ammonite.predef.UserPredef$.$anonfun$res_2$1(~/.ammonite/predef.sc:10)
        at ammonite.predef.UserPredef$.$anonfun$res_2$1$adapted(~/.ammonite/predef.sc:10)
        at scala.collection.immutable.List.foreach(List.scala:389)
        at ammonite.predef.UserPredef$.<init>(~/.ammonite/predef.sc:10)
        at ammonite.predef.UserPredef$.<clinit>(~/.ammonite/predef.sc)
        at ammonite.predef.UserPredef.$main(~/.ammonite/predef.sc)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$evalMain$1(Evaluator.scala:103)
        at ammonite.runtime.Evaluator$$anon$1.withContextClassloader(Evaluator.scala:110)
        at ammonite.runtime.Evaluator$$anon$1.evalMain(Evaluator.scala:103)
        at ammonite.runtime.Evaluator$$anon$1.$anonfun$processScriptBlock$1(Evaluator.scala:146)
        at ammonite.util.Catching.map(Res.scala:109)
        at ammonite.runtime.Evaluator$$anon$1.processScriptBlock(Evaluator.scala:144)
        at ammonite.interp.Interpreter.$anonfun$processScriptBlock$3(Interpreter.scala:255)
        at ammonite.util.Res$Success.flatMap(Res.scala:56)
        at ammonite.interp.Interpreter.$anonfun$processScriptBlock$2(Interpreter.scala:249)
        at ammonite.util.Res$Success.flatMap(Res.scala:56)
        at ammonite.interp.Interpreter.processScriptBlock(Interpreter.scala:246)
        at ammonite.interp.Interpreter.$anonfun$processModule$7(Interpreter.scala:308)
        at ammonite.interp.Interpreter.$anonfun$processCorrectScript$3(Interpreter.scala:432)
        at ammonite.util.Res$Success.flatMap(Res.scala:56)
        at ammonite.interp.Interpreter.compileRunBlock$1(Interpreter.scala:422)
        at ammonite.interp.Interpreter.$anonfun$processCorrectScript$9(Interpreter.scala:448)
        at ammonite.util.Res$Success.flatMap(Res.scala:56)
        at ammonite.interp.Interpreter.$anonfun$processCorrectScript$8(Interpreter.scala:447)
        at ammonite.util.Res$Success.flatMap(Res.scala:56)
        at ammonite.interp.Interpreter.loop$1(Interpreter.scala:445)
        at ammonite.interp.Interpreter.processCorrectScript(Interpreter.scala:495)
        at ammonite.interp.Interpreter.$anonfun$processModule$5(Interpreter.scala:310)
        at ammonite.util.Res$Success.flatMap(Res.scala:56)
        at ammonite.interp.Interpreter.processModule(Interpreter.scala:295)
        at ammonite.interp.Interpreter.$anonfun$new$2(Interpreter.scala:102)
        at ammonite.interp.Interpreter.$anonfun$new$2$adapted(Interpreter.scala:102)
        at ammonite.interp.PredefInitialization$.$anonfun$apply$7(PredefInitialization.scala:57)
        ... 13 more
```
leaned towards filing the issue here b/c things worked in `sbt console` with a minimal normal sbt project with
`build.sbt`
```scala
scalaVersion in ThisBuild := "2.12.2"
resolvers += "jitpack.io" at "https://jitpack.io"
libraryDependencies += "com.github.emanresusername"  % "squants-fx"          % "1e0e64ffbd"
```
`project/build.sbt`
```scala
Seq(
  "io.get-coursier" % "sbt-coursier" % "1.0.0-RC3"
).map(addSbtPlugin)
```
`project/build.properties`
```properties
sbt.version=0.13.15
```

**Comment by lihaoyi** [comment]
i guess the correct thing to do is ignore missing source dependencis

**Comment by emanresusername** [comment]
is that a knob i can turn in the `predef.sc` or does that require a code change here?

**Comment by lihaoyi** [comment]
I pushed a fix already, that should go out as an unstable version once I can make travis stop misbehaving

**Comment by lihaoyi** [comment]
Should be fixed in `1.0.0-RC4`

```scala
lihaoyi test$ amm
Loading...
Welcome to the Ammonite Repl 1.0.0-RC4
(Scala 2.12.2 Java 1.8.0_112)
If you like Ammonite, please support our development at www.patreon.com/lihaoyi
@ interp.repositories() ++= Seq(
      coursier.maven.MavenRepository("https://jitpack.io")
  )

@ interp.load.ivy("com.github.emanresusername"  % "squants-fx"          % "1e0e64ffbd")
https://jitpack.io/com/github/emanresusername/squants-fx/1e0e64ffbd/squants-fx-1e0e64ffbd.pom
  100.0% [##########] 993 B (2.5 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/root_2.12/1e0e64ffbd/root_2.12-1e0e64ffbd.p…
  100.0% [##########] 722 B (20.7 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/squants-fx_sjs0.6_2.12/1e0e64ffbd/squants-f…
  100.0% [##########] 1.8 KiB (53.8 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/squants-fx_2.12/1e0e64ffbd/squants-fx_2.12-…
  100.0% [##########] 1.4 KiB (45.3 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/1e0e64ffbd/squants-fx-1e0e64ffbd.jar
  100.0% [##########] 289 B (1.0 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/squants-fx_2.12/1e0e64ffbd/squants-fx_2.12-…
  100.0% [##########] 36.1 KiB (119.6 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/root_2.12/1e0e64ffbd/root_2.12-1e0e64ffbd.j…
  100.0% [##########] 266 B (860 B / s)
https://jitpack.io/com/github/emanresusername/squants-fx/squants-fx_sjs0.6_2.12/1e0e64ffbd/squants-f…
  100.0% [##########] 66.6 KiB (60.0 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/squants-fx_2.12/1e0e64ffbd/squants-fx_2.12-…
  100.0% [##########] 3.8 KiB (13.4 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/squants-fx_sjs0.6_2.12/1e0e64ffbd/squants-f…
  100.0% [##########] 3.8 KiB (13.7 KiB / s)
https://jitpack.io/com/github/emanresusername/squants-fx/root_2.12/1e0e64ffbd/root_2.12-1e0e64ffbd-s…
  100.0% [##########] 238 B (865 B / s)
Failed to load source dependenciesresusername/squants-fx/1e0e64ffbd/squants-fx-1e0e64ffbd-sources.ja…
  not found: https://jitpack.io/com/github/emanresusername/squants-fx/1e0e64ffbd/squants-fx-1e0e64ffbd-sources.jar

@
```

The warning printing is a bit weird - not sure why it's overlapping the coursier prints like that - but I'll fix that later. For now it works
